tests/runner.py

```diff
@@ -468,12 +468,14 @@ def tearDown(self):
         # Our leak detection will pick up *any* new temp files in the temp dir. They may not be due to
         # us, but e.g. the browser when running browser tests. Until we figure out a proper solution,
         # ignore some temp file names that we see on our CI infrastructure.
-        ignorable_files = [
+        ignorable_file_prefixes = [
           '/tmp/tmpaddon',
-          '/tmp/circleci-no-output-timeout'
+          '/tmp/circleci-no-output-timeout',
+          '/tmp/wasmer'
         ]
 
-        left_over_files = list(set(temp_files_after_run) - set(self.temp_files_before_run) - set(ignorable_files))
+        left_over_files = list(set(temp_files_after_run) - set(self.temp_files_before_run))
```

Comment:
While you are here maybe drop the `list()` here.. seems like keeping this as a set makes the most sense.

Reply:
Fixing.